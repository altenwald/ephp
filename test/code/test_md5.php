<?php

var_dump(md5("Would you like a green or red apple?"));
