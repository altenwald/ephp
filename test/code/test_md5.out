string(32) "ff67575ff98988e3b7d030f68569ee8c"
